apollo/src/models/onlineStatus.ts

```diff
@@ -0,0 +1,36 @@
+import { Schema, model } from 'mongoose';
+
+export enum OnlineStatus {
+  ONLINE = 'ONLINE',
+  OFFLINE = 'OFFLINE',
+}
+
+type OnlineStatusType = {
+  uid: String;
+  type: OnlineStatus;
+  last_seen: Date;
+};
+
+const schema = new Schema<OnlineStatusType>(
+  {
+    uid: {
```

Comment:
I think the `uid` attribute could be changed to `user_id` to match the database design, with type `mongoose.Types.ObjectId` ([Mongoose ObjectId](https://mongoosejs.com/docs/schematypes.html#objectids)), referencing `User` (you can take a look at [BE-33 PR](https://github.com/mobile-apcs-ntploc21/mobile-backend/pull/13/files#diff-a04818e26e6a66cb6e50851ac025fd0714626f1bc32d8f5374449ffd3b157a53R13)).

Reply:
I fixed it